### ISSUE TITLE
macOS: Can't type combining diacritics into query form

### ISSUE THREAD

**Issue by dlukes** [czcorpus/kontext#3724]
Typing `´` + `o` into the query form should normally result in `ó`, but on macOS, this behavior is broken. Instead, typing `´` inserts a `'`, and the following `o` doesn't insert anything but makes the cursor jump to the beginning of the line. This is also the case for the CQL query form of v0.13 but not the other query types, which means it's probably related to the use of `contenteditable=true`?

**Comment by tomachalek** [comment]
You're right. In KonText 0.15, both query types' inputs are based on `contenteditable`.

**Comment by dlukes** [comment]
FWIW, I noticed I'm also seeing this on Linux, though the behavior is slightly different: if I'm trying to type e.g. `ať` by pressing the sequence `a` `ˇ` `t`, then when I press `ˇ`, `ˇ` is inserted at that point in the text, and when I subsequently press `t`, the cursor jumps to the beginning of the text and `ť` is inserted. So the result is `ťaˇ`.

This is in the latest Firefox; in Chrome, it's also broken but less weird -- trying to type `ať` results in `aˇt`.
